tests/unit/Owncloud/OcisPhpSdk/EducationUserTest.php

```diff
@@ -39,4 +39,29 @@ public function testListEducationUsers()
         $user = $ocis->getEducationUserById("user-id");
         $this->assertInstanceOf(EducationUser::class, $user);
     }
+
+    /**
+     * @return void
+     */
+    public function testCreateEducationUsers()
```

Comment:
I don't think this test does anything useful as you are mocking the exact method that you are testing.
you would have to mock `EducationUserApi::createEducationUser` and then call `$ocis->createEducationUser()` for real